README.rst

```diff
@@ -184,7 +184,7 @@ You can find a quick demo of coala here:
 
 |asciicast|
 
-.. |asciicast| image:: https://asciinema.org/a/42968.png
+.. |asciicast| image:: https://raw.githubusercontent.com/coala/artwork/master/artwork/asciinema_demo/asciinema_demo.png
```

Comment:
It is working fine when I convert this rst to html. The target is still the name, no changes made there so it should work